pom.xml

```diff
@@ -214,6 +214,7 @@
     <!-- Overrides the version in the jboss-integration-platform-bom. A peer PR will be sent to jboss-integration-platform project.
          Please move this new version number there when possible -->
     <version.org.jboss.forge.roaster>2.19.4.Final</version.org.jboss.forge.roaster>
+    <version.org.jboss.concurrency-api>1.0.0.Final</version.org.jboss.concurrency-api>
```

Comment:
Yes, generally we should be using `version.<group-id>` where possible. If the property is ambiguous, `artifact-id` can be appended.